packages/sdk/src/sdk-blockchains/ethereum/bid.ts

```diff
@@ -138,12 +159,68 @@ export class EthereumBid {
 			multiple: collection.type === "ERC1155",
 			maxAmount: item.supply,
 			baseFee: await this.sdk.order.getBaseOrderFee(),
+			getConvertableValue: this.getConvertableValue,
+			convert: this.convertCurrency,
 			submit,
 		}
 	}
 
+	getConvertMap() {
+		return {
+			[`ETHEREUM:${this.sdk.balances.getWethContractAddress()}`]: "ETH",
+		}
+	}
+
+	private async getConvertableValue(
+		assetType: AssetType, value: BigNumberValue
+	): Promise<GetConvertableValueResult> {
+		if (!this.wallet) {
+			throw new Error("Wallet is undefined")
+		}
+		const convertMap = this.getConvertMap()
+
+		if (assetType["@type"] === "ERC20" && assetType["contract"] in convertMap) {
+			const walletAddress = await this.wallet.getAddress()
```

Comment:
can we add user address in arguments ? it's will give us more clear functionality and less dependencies such as wallet

Reply:
we can't do that
bid response should not depend on wallet